client/routes/unprotectedRoutes.tsx

```diff
@@ -4,13 +4,15 @@ import SignInPage from '@features/auth-pages/SignInPage';
 import SignUpPage from '@features/auth-pages/SignUpPage';
 import ForgotPasswordPage from '@features/auth-pages/ForgotPasswordPage';
 import PasswordUpdatePage from '@features/auth-pages/PasswordUpdatePage';
+import HomePage from '@features/home-page/HomePage';
 
 //TODO: replace palceholders with components when ready
-const authRoutes: RouteObject[] = [
+const unprotectedRoutes: RouteObject[] = [
   { path: 'userLogin', element: <SignInPage /> },
   { path: 'userSignup', element: <SignUpPage /> },
   { path: 'forgotPassword', element: <ForgotPasswordPage /> },
   { path: 'updatePassword', element: <PasswordUpdatePage /> },
+  { path: 'home-page', element: <HomePage /> },
```

Comment:
I think this route may be a duplicate, there is already a '/' route assigned to HomePage in 'publicRoutes.tsx'.

Reply:
I think I would change the 'home-page' string to 'home' and set '/' to redirect to '/home'.